set.mm

```diff
@@ -222585,6 +222586,18 @@ unital rings ( ~ df-ring ) and fields ( ~ df-field ).  (Contributed by
       ( cgrp wcel cmnd grpmnd syl ) ABDEBFECBGH $.
   $}
 
+  ${
+    grpcld.b $e |- B = ( Base ` G ) $.
+    grpcld.p $e |- .+ = ( +g ` G ) $.
+    grpcld.r $e |- ( ph -> G e. Grp ) $.
+    grpcld.x $e |- ( ph -> X e. B ) $.
+    grpcld.y $e |- ( ph -> Y e. B ) $.
+    $( Closure of the operation of a group.  (Contributed by SN,
+       29-Jul-2024.) $)
+    grpcld $p |- ( ph -> ( X .+ Y ) e. B ) $=
+      ( cgrp wcel co grpcl syl3anc ) ADLMEBMFBMEFCNBMIJKBCDEFGHOP $.
+  $}
+
```

Comment:
moved from mathbox: please update changes-set.txt accordingly

Reply:
since it is from my own mathbox I don't think this is necessary

Reply:
> since it is from my own mathbox I don't think this is necessary

Maybe not necessary, but usual practice (we had this discussion already before).

Reply:
> Maybe not necessary, but usual practice (we had this discussion already before).

At least, usual to you.  I'm fine either way, but this shouldn't bock this PR too long.